tools/build.js

```diff
@@ -126,8 +129,38 @@ await Promise.all([
   cp('src/build/templates', join(OUT_DIR, 'build/templates'), { recursive: true, force: true }),
 ])
 
+async function ensureNoRegionalBlobsModuleDuplicates() {
+  const REGIONAL_BLOB_STORE_CONTENT_TO_FIND = 'fetchBeforeNextPatchedIt'
+
+  const filesToTest = await glob(`${OUT_DIR}/**/*.{js,cjs}`)
+  const unexpectedModulesContainingFetchBeforeNextPatchedIt = []
+  let foundInExpectedModule = false
+  for (const fileToTest of filesToTest) {
+    const content = await readFile(fileToTest, 'utf-8')
+    if (content.includes(REGIONAL_BLOB_STORE_CONTENT_TO_FIND)) {
+      if (fileToTest.endsWith('run/regional-blob-store.cjs')) {
+        foundInExpectedModule = true
+      } else {
+        unexpectedModulesContainingFetchBeforeNextPatchedIt.push(fileToTest)
+      }
+    }
+  }
+  if (!foundInExpectedModule) {
+    throw new Error(
+      'Expected to find "fetchBeforeNextPatchedIt" variable in "run/regional-blob-store.cjs", but it was not found. This might indicate setup change that require bundling validation in "tools/build.js" to be adjusted.',
```

Comment:
nit

```suggestion
      'Expected to find "fetchBeforeNextPatchedIt" variable in "run/regional-blob-store.cjs", but it was not found. This might indicate a setup change that requires the bundling validation in "tools/build.js" to be adjusted.',
```